python/__init__.py

```diff
@@ -209,12 +209,13 @@ class Panda:
   FLAG_HONDA_NIDEC_ALT = 4
   FLAG_HONDA_RADARLESS = 8
 
-  FLAG_HYUNDAI_EV_GAS = 1
-  FLAG_HYUNDAI_HYBRID_GAS = 2
-  FLAG_HYUNDAI_LONG = 4
-  FLAG_HYUNDAI_CAMERA_SCC = 8
-  FLAG_HYUNDAI_CANFD_HDA2 = 8
-  FLAG_HYUNDAI_CANFD_ALT_BUTTONS = 16
+  # first byte is for max actuation torque
+  FLAG_HYUNDAI_EV_GAS = (1 << 8)
+  FLAG_HYUNDAI_HYBRID_GAS = (2 << 8)
+  FLAG_HYUNDAI_LONG = (4 << 8)
+  FLAG_HYUNDAI_CAMERA_SCC = (8 << 8)
+  FLAG_HYUNDAI_CANFD_HDA2 = (8 << 8)
+  FLAG_HYUNDAI_CANFD_ALT_BUTTONS = (16 << 8)
```

Comment:
why not leave these and use second byte for torque? seems cleaner

Reply:
oh that's what I did for the toyota one but I remember you flipped it